### ISSUE TITLE
evaluation模块的断言报错

### ISSUE THREAD

**Issue by 1kimli2** [Hope-Rita/PatCLS#4]
在evaluation模块这个断言也要注释掉吗？不然报错
 assert truth.shape == pred.shape

**Comment by 1kimli2** [comment]
但我看了后面调用的函数scores = get_metric(y_true=truth.detach().cpu(), y_pred=pred.detach().cpu())必须确保truth.shape，pred.shape这两个相等，所以这个断言应该没问题，那请问是为什么呢？怎么修改呢？
